### PR TITLE
add programmatic and manual alt tags

### LINKED ISSUE #233: Image alt tags
_part of a series of accessibility improvements//maintenance taeyoon asked me to work on, I'm going to sort through and make PR's for these where possible_

Few of the current pages on the site have images with alt tags. Particular places like class group photos, header images, etc. For specific people pages, I think that it would be nice to add programmatically built alt tags in the jekyll template.

### PR DESCRIPTION
**Pull request by Inkering**
additions of alt image information both manually and programmatically. Resolves #233

### CODE REVIEW

**Comment by Inkering** [comment]
image carousel display with alt text
*firefox devtools alt tag display*

I added a short snippet of code to enable adding alt tags to image carousel slides in markdown. The old slide specification still works, and then this is a progressively available option which is easy to use.

**Comment by Inkering** [comment]
- add front splash page

File: fall2016.md [hunk 1]
```diff
@@ -5,19 +5,30 @@ location: westbeth
...
+  - image: "/static/img/participate/bagels.jpg"
+    alt: "people discussing and eating bagels"
+  - image: "/static/img/participate/D4N.jpg"
+    alt: "Large art display of generative art, with code beside it"
+  - image: "/static/img/participate/poeticsciencefair.jpg"
+    alt: "People talking and discussing in a fair with laptops and poetic computation art"
+  - image: "/static/img/participate/goldenhour.jpg"
+    alt: "people working and talking on laptops during the golden hour lighting"
+  - image: "/static/img/participate/breadboard.jpg"
+    alt: "close up of electronics ntoes and prototyping breadboard"
```
Comment (solon) [inline fall2016.md:19]:
```suggestion
    alt: "close up of electronics notes and prototyping breadboard"
```

File: fall2018.md [hunk 2]
```diff
@@ -5,20 +5,32 @@ location: westbeth
...
+  - image: "/static/img/participate/allisonp.jpg"
+    alt: "Allison Parrish presenting to group of people"
+  - image: "/static/img/participate/fall15-hang.jpg"
+    alt: "three people talking in front of electrical diy equipment"
+  - image: "/static/img/participate/fall15-view.jpg"
+    alt: "sfpc space facing the courtyard, people are talking and looking at artworks"
+  - image: "/static/img/participate/fall15-sarah.jpg"
+    alt: "Sarah talking to a person in front of hung artwork"
+  - image: "/static/img/participate/fall16-katrina.jpg"
+    alt: "Katrina making hearing motion (hand cuped to ear) in front of welcome to the chatter box projection"
```
Comment (solon) [inline fall2018.md:19]:
```suggestion
    alt: "Katrina making hearing motion (hand cupped to ear) in front of welcome to the chatter box projection"
```

Comment (Inkering) [reply fall2018.md:19]: thanks for catching that!

File: participate/fall-2014.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: Participate 2014
 layout: default
 ---
 
-<img src="../img/SPCPicket.jpg" width="640">
+<img src="../img/SPCPicket.jpg" alt="people holding signs with text School for Poetic Computationon them" width="640">
```
Comment (solon) [inline participate/fall-2014.md:6]:
```suggestion
<img src="../img/SPCPicket.jpg" alt="people holding signs with text School for Poetic Computation on them" width="640">
```

File: poetichardware-spring2020.md [hunk 4]
```diff
@@ -28,10 +28,10 @@ Beginning with discrete electronic components such as resistors, LEDs and transi
 
 
 Past student projects. Photos by Tiri Kananuruk
-![](/static/img/poetichardware/L1550317.JPG) 
-![](/static/img/poetichardware/L1550461.JPG)
-![](/static/img/poetichardware/L1550634.JPG)
-![](/static/img/poetichardware/L1550652.JPG) 
+![student projects, people with palms together and led lights on hands](/static/img/poetichardware/L1550317.JPG)
```
Comment (solon) [inline poetichardware-spring2020.md:31]:
```suggestion
![student projects, people with palms together and LED lights on hands](/static/img/poetichardware/L1550317.JPG) 
```

File: poetichardware-spring2020.md [hunk 5]
```diff
@@ -28,10 +28,10 @@ Beginning with discrete electronic components such as resistors, LEDs and transi
 
 
 Past student projects. Photos by Tiri Kananuruk
-![](/static/img/poetichardware/L1550317.JPG) 
-![](/static/img/poetichardware/L1550461.JPG)
-![](/static/img/poetichardware/L1550634.JPG)
-![](/static/img/poetichardware/L1550652.JPG) 
+![student projects, people with palms together and led lights on hands](/static/img/poetichardware/L1550317.JPG) 
+![student projects, box with wire and led spelling out hello world](/static/img/poetichardware/L1550461.JPG)
```
Comment (solon) [inline poetichardware-spring2020.md:32]:
```suggestion
![student projects, box with wire and LED spelling out hello world](/static/img/poetichardware/L1550461.JPG)
```

File: poetichardware-spring2020.md [hunk 6]
```diff
@@ -60,19 +60,19 @@ Past student projects. Photos by Tiri Kananuruk
...
 
 Taylor Levy [pron. tey-ler] is an artist & designer with a penchant for taking things apart, understanding how they work, and then putting them back together in a way that exposes their inner workings.The results take on a variety of forms from low-tech electronic sculpture to high-tech software & other executions. She has work on view at The Leonardo Museum of Science and Technology and was a resident at Fabrica Interactive in Treviso, Italy. She is an alumna of MIT Media Lab, ITP at NYU, and Vassar College.
 
-![](/static/img/poetichardware/symbiosis.gif) 
+![device by Sebastian Moralest](/static/img/poetichardware/symbiosis.gif)
```
Comment (solon) [inline poetichardware-spring2020.md:71]:
```suggestion
![device by Sebastian Morales](/static/img/poetichardware/symbiosis.gif) 
```

File: recreatingthepast-spring2020.md [hunk 7]
```diff
@@ -81,16 +94,16 @@ This class is not right for you if:
 -  You are interested in a more skills based, less theoretical approach to learning creative code
 -  You are not interested in research or engaging with artworks from the past
 
-![](/static/img/recreating/recreatingthepast3.png) 
+![digital graphic with peoples names in perspective](/static/img/recreating/recreatingthepast3.png)
```
Comment (solon) [inline recreatingthepast-spring2020.md:97]:
```suggestion
![digital graphic with people's names in perspective](/static/img/recreating/recreatingthepast3.png) 
```

File: spring2018.md [hunk 8]
```diff
@@ -5,19 +5,32 @@ location: westbeth
...
+  - image: "/static/img/participate/allisonp.jpg"
+    alt: "Allison Parrish presenting to group of people"
+  - image: "/static/img/participate/fall15-hang.jpg"
+    alt: "three people talking in front of electrical diy equipment"
+  - image: "/static/img/participate/fall15-view.jpg"
+    alt: "sfpc space facing the courtyard, people are talking and looking at artworks"
+  - image: "/static/img/participate/fall15-sarah.jpg"
+    alt: "Sarah talking to a person in front of hung artwork"
+  - image: "/static/img/participate/fall16-katrina.jpg"
+    alt: "Katrina making hearing motion (hand cuped to ear) in front of welcome to the chatter box projection"
```
Comment (solon) [inline spring2018.md:19]:
```suggestion
    alt: "Katrina making hearing motion (hand cupped to ear) in front of welcome to the chatter box projection"
```

File: summer16short.md [hunk 9]
```diff
@@ -5,13 +5,21 @@ location: westbeth
...
 
-  - "/static/img/participate/SFPC2-weekSummer.jpg"
-  - "/static/img/participate/wallbw.jpg"
-  - "/static/img/participate/flower.jpg"
-  - "/static/img/participate/door.jpg"
-  - "/static/img/participate/havefun.jpg"
-  - "/static/img/participate/poeticsciencefair.jpg"
-  - "/static/img/participate/notebook.jpg"
+  - image: "/static/img/participate/SFPC2-weekSummer.jpg"
+    alt: "SFPC 2 week summer session text photoshopped onto nytimes front page"
```
Comment (solon) [inline summer16short.md:9]:
```suggestion
    alt: "SFPC 2 week summer session text photoshopped onto NY Times front page"
```

File: summer16short.md [hunk 10]
```diff
@@ -5,13 +5,21 @@ location: westbeth
...
+  - image: "/static/img/participate/wallbw.jpg"
+    alt: "sfpc code poetry wall art with polaroids of sfpc students and people"
+  - image: "/static/img/participate/flower.jpg"
+    alt: "books next to flowers, the books are conditional design workbook, speaking code, and the art of the art assignment"
+  - image: "/static/img/participate/door.jpg"
+    alt: "sfpc door with signs and sticks all over it, its a pull and push door"
```
Comment (solon) [inline summer16short.md:15]:
```suggestion
    alt: "sfpc door with signs and sticks all over it, it's a pull and push door"
```

File: _classes/summer2016show/index.html [hunk 11]
```diff
@@ -149,7 +149,7 @@ <h1 class="title">Guhong Min</h1>
         <div class="container">
           <h1 class="title">Hélène Martin</h1>
           <figure class="image">
-            <img src="assets/images/helene.gif" alt="Hélène Martin">
+            <img src="assets/images/helene.gif" alt="Hélène Martin's banner, representation of the statue of liberty with red led attached">
```
Comment (solon) [inline _classes/summer2016show/index.html:152]:
```suggestion
            <img src="assets/images/helene.gif" alt="Hélène Martin's banner, representation of the statue of liberty with red LED attached">
```

**Review by solon — COMMENTED** [review]
Thanks so much @Inkering for this!

**Pull request merged**